caddy.go

```diff
@@ -836,7 +839,7 @@ var (
 	instancesMu sync.Mutex
 )
 
-const (
+var (
```

Comment:
[ Quoting <notifications@github.com> in "Re: [mholt/caddy] Implement missing..." ]
>> @@ -836,7 +839,7 @@ var (
>>  	instancesMu sync.Mutex
>>  )
>>
>> -const (
>> +var (
>
>Ohh, from your main function, not your plugin, I see... that should work. Okay.

Yep! :-) (sorry if that wasn't clear)